hw/ip/soc_ctrl/data/soc_ctrl.hjson

```diff
@@ -52,12 +52,21 @@
     { name:     "USE_SPIMEMIO",
       desc:     "Spi Module Select Value - Used to decide whether to use the SPI from Yosys or OpenTitan",
       resval:   "0x1"
-      swaccess: "ro",
+      swaccess: "rw",
       hwaccess: "hrw",
       fields: [
         { bits: "0", name: "USE_SPIMEMIO", desc: "Spi Select Reg" }
       ]
     }
+    { name:     "ENABLE_SPI_SEL",
+      desc:     "Enable Spi module selection from software",
+      resval:   "0x0"
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0", name: "ENABLE_SPI_SEL", desc: "Enable Spi Select Reg" }
```

Comment:
why do we want to enable the previous bit? is that what you want to do?

Reply:
I implemented this because at the beginning I used the spiflash crt0 that is using the spimemio to copy the data from flash to ram, so I needed a way to switch between the two SPI from software. Now I only rely on the OT SPI host, so this could be removed, but I believe it is still interesting to be able to switch from one to the other after the booting is done.